go.mod

```diff
@@ -0,0 +1,7 @@
+module atomized.org/xpod
```

Comment:
This should be a valid git URL, if you ever want this to be importable by anything else.